test/jasmine/tests/scattermapbox_test.js

```diff
@@ -365,6 +365,23 @@ describe('scattermapbox convert', function() {
         expect(actualText).toEqual(['A', 'B', 'C', 'F', '']);
     });
 
+    it('for lines traces with trailing gaps', function() {
```

Comment:
same comment as https://github.com/plotly/plotly.js/pull/1414#discussion_r103071216 - is this an intentional pattern now?